kolibri_explore_plugin/collectionviews.py

```diff
@@ -430,7 +436,7 @@ def _remotechannelimport(user, channel_id):
             "channel_name": "foo",
         },
     )
-    job_id = remotechannelimport.enqueue(job=job)
+    job_id = remotechannelimport.enqueue(job=job, priority=Priority.HIGH)
```

Comment:
This change shouldn't be needed: the `remotechannelimport` task has priority=Priority.HIGH by default.

```suggestion
    job_id = remotechannelimport.enqueue(job=job)
```